trinity/chains/__init__.py

```diff
@@ -56,6 +65,50 @@
 )
 
 
+def validate_genesis(chain_config: ChainConfig) -> (BlockHeader, int):
+    """
+    Checks the genesis configuration file provided and ensures that it is valid.
+    """
+    if not os.path.exists(chain_config.genesis):
+        raise MissingPath(
+            "The base chain genesis configuration file does not exist: `{0}`".format(
+                chain_config.genesis,
+            ),
+        )
+
+    with open(chain_config.genesis, 'r') as genesis_config:
+        raw_genesis = genesis_config.read()
+        genesis = json.load(raw_genesis)
```

Comment:
These two lines should be replaced with `genesis = json.load(genesis_config)`.  No need to `read()` the file object.